modules/ROOT/pages/indexes/search-performance-indexes/using-indexes.adoc

```diff
@@ -0,0 +1,1013 @@
+== Over-indexing: considerations and solutions
+
+Search-performance indexes can significantly improve query performance.
+They should, however, be used judiciously for the following reasons:
+
+* *Storage space*: because each index is a secondary copy of the data in the primary database, each index essentially doubles the amount of storage space occupied by the indexed data.
+* *Slower write queries*: adding indexes impacts the performance of write queries.
+This is because indexes are updated with each write query. If a system needs to perform a lot of writes quickly, it may be counterproductive to have an index on the affected data entities.
+In other words, if write performance is crucial for a particular use case, it may be beneficial to only add indexes where they are necessary for read-query purposes.
+
+As a result of these two points, deciding what to index (and what not to index) is an important and non-trivial task.
+
+[[tracking-index-use]]
+=== Keeping track of index-use: `lastRead`, `readCount`, and `trackedSince`
+
+Unused indexes take up unnecessary storage space and it may be beneficial to remove them.
+Knowing which indexes are most frequently used by the queries against a database can, however, be difficult.
+As of Neo4j 5.8, there are three relevant columns returned by the xref:indexes/search-performance-indexes/managing-indexes.adoc#list-indexes[`SHOW INDEX`] command which can help identify redundant indexes:
+
+* *`lastRead`*: returns the last time the index was used for reading.
+* *`readCount`*: returns the number of read queries issued to the index.
+* *`trackedSince`* returns the time when usage statistics tracking started for an index.footnote:[The `trackedSince` column is not part of the default return columns for the `SHOW INDEXES` command. To return this and all other non-default columns, use `SHOW INDEXES YIELD *`.
+For more information, see xref:indexes/search-performance-indexes/managing-indexes.adoc#listing-indexes-result-columns[Managing search-performance indexes -> Result columns for listing indexes].]
+
+To return these values (along with other relevant information) for the indexes in a database, run the following query:
+
+.Query to identify redundant indexes
+[source,cypher]
+----
+SHOW INDEX YIELD name, type, entityType, labelsOrTypes, properties, lastRead, readCount, trackedSince
+----
+
+If any unused indexes are identified, it may be beneficial to delete them using the xref:indexes/search-performance-indexes/managing-indexes.adoc#drop-indexes[`DROP INDEX`] command.
+
+[[summary]]
+== Summary
+
+* Range indexes can be used to solve most predicates.
+
+* Text indexes are used over range indexes for `CONTAINS` and `ENDS WITH` predicates on `STRING` properties, and if the queried `STRING` properties exceed 8 kb.
+
+* Point indexes are used when queries filter on distances and bounding boxes.
+
+* Token lookup indexes are not defined in this order since they never solve the same predicates as other indexes.
```

Comment:
What does "not defined in this order" mean? :thinking:

Reply:
Yes, that is not clear. Let's just say "only solve node label and relationship type expressions, and cannot solve any predicates involving properties."